tests/acceptance/config/behat.yml

```diff
@@ -48,6 +48,20 @@ default:
         - OccContext:
         - FeatureContext: *common_feature_context_params
 
+    webUIPasswordAddUser:
+      paths:
+        - %paths.base%/../features/webUICreateUser
```

Comment:
There is no folder ``webUICreateUser``
https://drone.owncloud.com/owncloud/password_policy/309/458
```
No scenarios
No steps
0m0.39s (31.43Mb)
Information: no matching scenarios were found.
```
no tests are being run.